src/handlebars/installation.handlebars

```diff
@@ -2,57 +2,72 @@
 
 <main class="grey-bg">
 
-  <h1 class="large-title">Installation</h1>
-
-  <h2><a href="./migration.html" target="_blank">Migration Guide</a></h2>
-
-  <div class="align-center">
-    <div class="btn-container">
-      <form id="jdk-selector" class="btn-form">
-        <h3>1. Choose a Version</h3>
-      </form>
-      <form id="jvm-selector" class="btn-form">
-        <h3>2. Choose a JVM</h3>
-      </form>
+  <h1 class="margin-bottom large-title">Installation</h1>
+
+  <div class="align-left support">
+    <div class="margin-bottom">
+      <p>AdoptOpenJDK binaries are available for download in the following types of installation package:
+        <ul>
+          <li><a href="#archives">Archive files</a></li>
+          <li><a href="#installers">Installers</a></li>
+        </ul>
+
+    <p>The binaries are supported on the architectures and operating systems listed in <a href="./supported_platforms.html">Supported Platforms</a>. If you're migrating
+    to AdoptOpenJDK, you can learn about the differences between Oracle JDK and AdoptOpenJDK in our <a href="./migration.html" target="_blank">Migration Guide</a> and any
+    steps that you might need to take. For example, how to use IcedTea-Web as an alternative to Webstart.</p>
+
+    <div>
+      <br>
+      <hr>
```

Comment:
Could apply some CSS to the hr tag or the div to create the vertical space as opposed to br tags?

Reply:
Applied class="margin-bottom" to the &lt;div&gt;